plugins/collection-log-luck

```diff
@@ -0,0 +1,3 @@
+repository=https://github.com/peanubnutter/collection-log-luck.git
+commit=c7f05a80b8195ac2633ecf579aaf0c3e8d173f94
+warning=This plugin pulls data from collectionlog.net, a 3rd party server not controlled or verified by the RuneLite developers, and which can access your IP address.
```

Comment:
```suggestion
warning=This plugin submits your username and IP address to a server not controlled or verified by the RuneLite developers.
```

Reply:
The original collection log plugin has that language, but this plugin does not POST any data - only GETs data, so it doesn't really "submit" anything, especially not username. Thoughts?